ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml

```diff
@@ -1015,6 +1015,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )sro-e2e-master,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
can you manually edit this file and turn off `always_run`?
with clusters being resource intensive, we (I) think it's better to keep the testing "human controlled" ;-)

the following `make jobs` will take care not to override this setting